packages/web/utils/trpc.ts

```diff
@@ -26,7 +29,27 @@ const getBaseUrl = () => {
 /** A set of type-safe react-query hooks for your tRPC API. */
 export const api = createTRPCNext<AppRouter>({
   config() {
+    const localStoragePersister = createSyncStoragePersister({
+      storage: typeof window !== "undefined" ? window.localStorage : undefined,
+      serialize: (client) => superjson.stringify(client),
+      deserialize: (cachedString) => superjson.parse(cachedString),
+    });
+
+    const queryClient = new QueryClient({
+      defaultOptions: {
+        queries: {
+          cacheTime: 1000 * 60 * 60 * 24, // 24 hours
```

Comment:
Wouldn't this cache time affect other queries? This could unintentionally impact prices

Reply:
This one is a global config yes, but it doesn't impact other queries in a bad way, because it's only the cacheTime:
https://tanstack.com/query/v4/docs/framework/react/plugins/persistQueryClient#how-it-works

By default all queries on tanstack, unless otherwise specified, have a[ `staleTime` equal to zero.](https://tanstack.com/query/v4/docs/framework/react/guides/caching#basic-example)
This means that when we load the page, if we find cached data we show it right away, but since the `staleTime` is equal to zero in the background revalidation of the data will take place and it will then be updated, in short as long as we don't have fresh data for up to 24 hours it shows the cached one.

It may not be a problem for the prices, however, it is definitely to be paid attention to during testing